domain-model/src/main/java/uk/gov/hmcts/cmc/domain/models/claimantresponse/ResponseRejection.java

```diff
@@ -19,21 +20,29 @@
 
     private final YesNoOption freeMediation;
 
+    @Size(max = 30, message = "may not be longer than {max} characters")
```

Comment:
okay

Reply:
I did a test, its not printing the name of the field. Strange. Validator is not printing the name of the field. may be while runtime it does. 
Better to do a component test and see.

Reply:
`@Test
    public void testAmountRow(){
        ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
        Validator validator = factory.getValidator();
        EvidenceRow evidenceRow = new EvidenceRow("1", EvidenceType.CONTRACTS_AND_AGREEMENTS,
            "dsdfsdfsdfsdsfsdfsdfsdf");

        Set<ConstraintViolation<EvidenceRow>> violations = validator.validate(evidenceRow);
        violations.stream().forEach(v -> System.out.println(v.getMessage()));
    }`

This returned 
```13:07:38.522 [Test worker] DEBUG org.hibernate.validator.resourceloading.PlatformResourceBundleLocator - ContributorValidationMessages not found.
13:07:38.524 [Test worker] DEBUG org.hibernate.validator.resourceloading.PlatformResourceBundleLocator - org.hibernate.validator.ValidationMessages found.
may not be longer than 9 characters
```

Reply:
Please check the code. The  BeanValidator.java (Test Util Code) does this for you just for test case sake. But in the running code it wont. So what you mentioned above is wrong. It will not give the field name at all.

Reply:
@KiranVarma02 note

Reply:
@KiranVarma02 the BeanValidation is in src/test :-)